### DISCUSSION TITLE
Storer for Redis/PostgreSQL DB - would you be interested in open sourcing yours? (how much $ would it take to implement)

### DISCUSSION THREAD

**Discussion by wkoszek** [aarondl/authboss#323] [General]
Hi folks. Other than https://github.com/rivo/users it appears like https://github.com/volatiletech/authboss is the only solution for solving authorization for Webapps in Golang. I wonder what would it take to get `authboss` to the point where with 10 lines of the code we could get something safe, that works, out-of-the-box.

Several folks in Issues appear to have implemented practical storers that resulted in some questions.  I wonder if anyone would be interested in open sourcing theirs?

I think having storers for PostgreSQL / Redis would bump the `authboss` adoption because it's nice to have something ready-to-use.

I could pitch some $ for this project, so if you're interesting, ping me with a quote. [specs are simple: web app uses 1 separate PostgreSQL DB with schema "authboss" and "users" table]

**Comment by aarondl** [comment]
Thanks for doing this @wkoszek. Hopefully we get some nice implementations coming out of this. We really could use a whole host of implementations for cookie/sessions, renderers, router adapters, and storers. 👍🏽

**Reply by wkoszek** [reply]
I'm wondering how to get more folks to be more active here. Are there any companies that you think we could hit and get some attention or $.

Would you be interested in some contracted work for doing this? :)

**Reply by aarondl** [reply]
Hi @wkoszek not personally interested in doing any contract bits for this. It's pretty straightforward stuff though, any reasonably competent developer could tackle it.
